nixos/services/sensu/client.nix

```diff
@@ -105,6 +105,15 @@ let
     };
   };
 
+  systemdUnitCheckOptions = {
```

Comment:
typo in commit message:
*seen* -> *seem* in
> which doesn't seen to
> work properly anymore

Also: *separated checks* -> *separate checks*, as their defining factor is not that they're segmented (separated) from each other but standalone checks (separate).